patterns/pagination.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Title: Default Pagination 
+ * Slug: course/pagination
+ * Inserter: yes
+ */
+?>
+
+<!-- wp:group {"layout":{"inherit":false,"type":"constrained"}} -->
+<div class="wp-block-group">
+  <!-- wp:query-pagination {"layout":{"type":"flex","justifyContent":"space-between"}} -->
+
+  <!-- wp:query-pagination-numbers /-->
+
+  <!-- wp:query-pagination-next {"label":"NEXT"} /-->
```

Comment:
If we're transforming this to uppercase later in `theme.json`, does this need to be capitalized here? I'm just thinking of translators who may think it's weird when they see this string is in all caps.

Reply:
Fixed here 0a275ef8803fc965c911b3a679e83d66736fe804